enclave_manager/avalon_enclave_manager/enclave_manager.py

```diff
@@ -160,6 +160,131 @@ def manager_on_boot(self, kv_helper):
         # End of for-loop
 
     # -----------------------------------------------------------------
+    def process_work_orders_sync(self, kv_helper, process_wo_id):
```

Comment:
`process_work_orders_sync()` and `process_work_orders()` function has lot of duplicate code. Need to address it. @manju956 Could you review this PR

Reply:
Sure, will review.